t/configuration-loading-boot-with-config.t

```diff
@@ -1,14 +1,11 @@
 use lib 't';
-use TestAPIcast 'no_plan';
+use TestAPIcastBlackbox 'no_plan';
 
-$ENV{TEST_NGINX_HTTP_CONFIG} = "$TestAPIcast::path/http.d/init.conf";
 $ENV{APICAST_CONFIGURATION_LOADER} = 'boot';
-$ENV{THREESCALE_CONFIG_FILE} = 't/servroot/html/config.json';
+#$ENV{TEST_NGINX_NO_CLEAN} = 1;
```

Comment:
Commented.

Reply:
Well spotted. 👍

Reply:
Fixed in https://github.com/3scale/apicast/pull/496/commits/19a3bb85984ef0441779018d6bf4151eb2110aa7 and will be squashed.